packages/components/icon/scripts/export-svg.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const { EOL } = require('os');
+const fs = require('fs').promises;
+const { camelize } = require('../../../../scripts/helpers');
+
+/**
+ *  Read all svg files in an assets/icons directory and generate index.js file with ES module exports
+ */
+(async function () {
+	const componentDir = path.join(process.cwd(), '/lib/assets/icons');
+	console.log(componentDir);
```

Comment:
Can you please either remove it or just put more text next to it to clarify what is it in the log?